### PR TITLE
feat(proposal page): update ui of the proposal page

### PR DESCRIPTION
**Pull request by stee1house**

### CODE REVIEW

File: features/proposal/components/VoterDetailsCard/tests/VoterDetailsCard.spec.tsx [hunk 1]
```diff
@@ -9,22 +9,6 @@ jest.mock('components/Icon', () => {
 });
 
 describe('VoterDetailsCard', () => {
-  // it('Should render component', () => {
```
Comment (AlexeyPonomarenko) [inline features/proposal/components/VoterDetailsCard/tests/VoterDetailsCard.spec.tsx:12]: Please, fix it)

File: features/proposal/components/VoteCollapsableList/VoteCollapsableList.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  data: VoterDetail[];
+}
+
+export const VoteCollapsableList: FC<VoteCollapsableListProps> = ({ data }) => {
+  const votesGroups: { [key: string]: VoterDetail[] } = useMemo(
+    () => groupBy(data, 'groups'),
+    [data]
+  );
+
+  if (!data?.length) {
```
Comment (AlexeyPonomarenko) [inline features/proposal/components/VoteCollapsableList/VoteCollapsableList.tsx]: we can use lodash `isEmpty` which is more laconic

File: features/proposal/components/VoteCollapsableHeader/VoteCollapsableHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    [votes]
+  );
+
+  return (
+    <div
+      tabIndex={0}
+      role="button"
+      className={cn(styles.header, { [styles.open]: state })}
+      onClick={() => setToggle(!state)}
+      onKeyPress={() => undefined}
```
Comment (AlexeyPonomarenko) [inline features/proposal/components/VoteCollapsableHeader/VoteCollapsableHeader.tsx:37]: I consider we should call setToggle here or provide '-1' to tabIndex

File: features/proposal/components/VoterDetailsCard/tests/VoterDetailsCard.spec.tsx (hunk 1 above)
Comment (stee1house) [reply features/proposal/components/VoterDetailsCard/tests/VoterDetailsCard.spec.tsx:12]: I removed only unused, commented code from the test :)

File: features/proposal/components/VoteCollapsableList/VoteCollapsableList.tsx (hunk 2 above)
Comment (stee1house) [reply features/proposal/components/VoteCollapsableList/VoteCollapsableList.tsx]: ok, I will fix

File: features/proposal/components/VoteCollapsableHeader/VoteCollapsableHeader.tsx (hunk 3 above)
Comment (stee1house) [reply features/proposal/components/VoteCollapsableHeader/VoteCollapsableHeader.tsx:37]: Ok, I will fix it.

**Review by AlexeyPonomarenko — APPROVED** [review]

**Pull request merged**